ci/fireci/fireci/commands.py

```diff
@@ -61,3 +63,38 @@ def smoke_tests(app_build_variant, test_apps_dir):
             os.path.join(cwd, 'build', 'm2repository')),
         workdir=location,
     )
+
+
+@click.argument('issue_number', required=True)
+@click.argument('repo_name', required=True)
+@click.argument('auth_token', required=True)
+@ci_command()
+def api_information(auth_token, repo_name, issue_number):
+  """Comments the api information on the pr"""
+
+  gradle.run('apiInformation')
+  gradle.run('generateApiTxtFile')
+  dir_suffix = 'build/apiinfo'
+  comment_string = ""
+  for filename in os.listdir(dir_suffix):
+    subproject = filename
+    formatted_output_lines = []
+    with open(os.path.join(dir_suffix, filename), 'r') as f:
+      outputlines = f.readlines()
+      for line in outputlines:
+        if 'error' in line:
+          formatted_output_lines.append(line[line.find('error:'):])
+    if formatted_output_lines:
+      comment_string += 'The public api surface has changed for the subproject {}:\n'.format(subproject[len('subproject:'):])
+      comment_string += ''.join(formatted_output_lines)
+      comment_string += '\n'
+  if comment_string:
+    comment_string += ('Please update the api.txt files for the subprojects being affected by this change '
+      'with the files present in the artifacts directory. Also perform a major/minor bump accordingly.\n')
+  else:
+    comment_string = 'No project\'s public api surface was changed due to this PR.\n'
```

Comment:
Do we want to post the comment unconditionally? It might make sense to not post it at all if everything looks ok, wdyt?